backends/module.mk

```diff
@@ -146,14 +146,25 @@ MODULE_OBJS += \
 	graphics/surfacesdl/surfacesdl-graphics.o \
 	mixer/sdl/sdl-mixer.o \
 	mutex/sdl/sdl-mutex.o \
-	plugins/sdl/sdl-provider.o \
 	timer/sdl/sdl-timer.o
 
+ifdef KOLIBRIOS
+MODULE_OBJS += plugins/kolibrios/kolibrios-provider.o
+endif
+
+ifndef RISCOS
+ifndef KOLIBRIOS
+MODULE_OBJS += plugins/sdl/sdl-provider.o
+endif
+endif
+
 # SDL 2 removed audio CD support
 ifndef USE_SDL2
+ifndef KOLIBRIOS
```

Comment:
Same thing, with the stubs, these modifications could go away

Reply:
Done